server/controllers/reviewDataConroller/reviewDataController.js

```diff
@@ -0,0 +1,49 @@
+const router = require('express').Router();
```

Comment:
I notice you are using the Express Router. What is the benefit of using this over using app.get, app.post, etc. directly?

Reply:
Separation of concerns to avoid cluttered code in the main server.js file. I don't think there are benefits in terms of performance, but there definitely are better opportunities for well-structured code using the router instead. 

https://scotch.io/tutorials/learn-to-use-the-new-router-in-expressjs-4
https://stackoverflow.com/questions/28305120/differences-between-express-router-and-app-get